x-pack/spec/modules/azure/filters/azure_event_spec.rb

```diff
@@ -9,6 +9,10 @@
 
 
 describe LogStash::Filters::AzureEvent do
+  before(:all) do
```

Comment:
It's required because the spec require `logstash/devutils/rspec/spec_helper` and we need the constant `LogStash::Config::PipelineConfig` to be assigned to the Java class else the tests fails

Reply:
to avoid side effect on other parts we could add an:
```ruby
after(:all) do
  LogStash::Config::PipelineConfig = nil
end
```

WDYT?

Reply:
that would be slightly weird :smiley_cat: but I get your point to `remove_const :PipelineConfig`
shouldn't be necessary if we agree on the compatibility `pipeline_config.rb` as suggested https://github.com/elastic/logstash/pull/11824#discussion_r414589573 ... we can than, for legacy spec reasons, just `require 'logstash/config/pipeline_config.rb'` as before.

Reply:
done